src/mpsse.rs

```diff
@@ -471,24 +118,15 @@ pub trait FtdiMpsse: FtdiCommon {
     ///
     /// let mut ft = Ft4232h::with_serial_number("FT4PWSEOA")?;
     /// ft.initialize_mpsse_default()?;
-    /// ft.set_clock(100_000)?;
+    /// ft.set_mpsse_clock(100_000)?;
     /// # Ok::<(), std::boxed::Box<dyn std::error::Error>>(())
     /// ```
-    fn set_clock(&mut self, frequency: u32) -> Result<(), TimeoutError> {
-        let (value, divisor) = clock_divisor(Self::DEVICE_TYPE, frequency);
-        debug_assert!(value <= 0xFFFF);
-
-        let mut buf: Vec<u8> = Vec::new();
-
-        if let Some(div) = divisor {
-            buf.push(div.into());
-        };
-
-        buf.push(MpsseCmd::SetClockFrequency.into());
-        buf.push((value & 0xFF) as u8);
-        buf.push(((value >> 8) & 0xFF) as u8);
+    fn set_mpsse_clock(&mut self, frequency: u32) -> Result<(), TimeoutError> {
```

Comment:
What was the purpose for the rename?
Ideally breaking changes can be kept to a minimum, but I am happy to rename it if there is a good reason.

Reply:
No real purpose. So reverting it here and in ftdi-mpsse crate. Fixed: new commit added to this PR.